### DISCUSSION TITLE
Draggable cards

### DISCUSSION THREAD

**Discussion by odysseyskate** [Budibase/budibase#4899] [Help]
I have a list of cards on a page & I want the user to be able to click & drag them around so they are ordered differently. I want the order they place the cards in to then be saved.

Is this possible?

**Comment by mjashanks** [comment]
Unfortunately not possible at the moment

**Reply by odysseyskate** [reply]
What about a button on the page to move a card up or down?

**Reply by mjashanks** [reply]
Yep, you can do this, but it will require you to write some javascript. For example, if you want to move a record "up", then you would need to:

1. Create an "Order" column on your table/datasource
On Button click..
2. find the record above the current one
3. Decrease it's order and save
4. Increase the current item's order and save it

**Comment by puntaschoice** [comment]
I'm also looking for a similar solution, or even a drop down list that the cards or items can be moved (grab handles on left of card column), the ability to add with a plus (at the bottom of the list or some other UI) from a datasource / provider / condition.

This has sent me off looking at other low code's but I would much prefer to pursue in Budibase.
Any other suggestions on a way to get something similar? The solution above with some JS sounds a tad scary.

Many thanks
